include/unk_020508D4.h

```diff
@@ -16,7 +16,7 @@ void sub_020509D4(FieldSystem * param0);
 BOOL sub_020509DC(FieldSystem * param0);
 void sub_02050A38(TaskManager * param0, const OverlayManagerTemplate * param1, void * param2);
 FieldSystem * TaskMan_FieldSystem(TaskManager * param0);
```

Comment:
Should this function also be renamed to match the `TaskManager` prefix?

Reply:
oops yes lemme fix it